commit 78790fd6299093d83ffab3fa344bf1259209d9d3
Author: William Kent <wjk@sunsol.me>
Date:   Wed Oct 31 16:42:29 2018 -0400

    Revert "Revert "apply fix_ipsec_compilation.patch""
    
    This reverts commit 1d5146069d8b802621ea39b21892c060a678a730.

diff --git a/bsd/net/if_ipsec.c b/bsd/net/if_ipsec.c
index 0984205..54391db 100644
--- a/bsd/net/if_ipsec.c
+++ b/bsd/net/if_ipsec.c
@@ -385,7 +385,11 @@ ipsec_interface_needs_netagent(ifnet_t interface)
 		return (FALSE);
 	}
 
+#if ENABLE_NEXUS
 	return (pcb->ipsec_needs_netagent == true);
+#else
+	return false;
+#endif
 }
 
 static errno_t
